RELEASE_NOTES.md

```diff
@@ -1,8 +1,9 @@
 # Release Notes
 
-## 5.13.4 - tbd
+## 5.13.5 - 2019-05-01
 
-* tbd
+* BUGFIX: `Shell.mv` now has the correct documented behavior (5.13.3 was still broken) - https://github.com/fsharp/FAKE/pull/2309
+* BUGFIX: `Shell.rename` now works for files and directories as documented - https://github.com/fsharp/FAKE/pull/2309
```

Comment:
@fangyi-zhou Please feel free to review this commit.

Reply:
LGTM